### PR TITLE
Add Windows nanoserver builder

### PR DESCRIPTION
**Pull request by jromero**
### TODO
- [ ] Add building and deployment of builder to CI

### CODE REVIEW

**Comment by micahyoung** [comment]
Hey @jromero . Would you mind if we named the builder `nanoserver-1809` instead of `nanoserver`? The catch is that these stacks/builders aren't as versatile as say `alpine` - they can only be used for `pack build` (for now) on a Windows 10 1809 daemon (aka Windows 10 Enterprise 2019 or Windows Server 1809).

There's some more complex ways to work around needing `1809` expressed everywhere with manifest lists but I feel that starts to exceed the goals of easy-to-understand sample.

**Comment by jromero** [comment]
@micahyoung completely agree. 1809 is included in the tag name as currently implemented in the PR. Am I missing something?

**Comment by micahyoung** [comment]
Oh I'm so sorry, I missed this was a PR 🤦. I'll properly review

File: builders/nanoserver-1809/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Sample Nano Server 1809 Builder
+
+### Prerequisites
+* [Pack](https://buildpacks.io/docs/install-pack/)
+
```
Comment (micahyoung) [inline builders/nanoserver-1809/README.md:5]: Maybe a `* Windows 10 Enterprise 2019 or Windows Server 1809`? That might also be implied by the title of the README

**Comment by micahyoung** [comment]
Oh, sorry for the spam @jromero ... I realize this is still your draft. Please disregard the comments, I just put them here for our story.

**Comment by jromero** [comment]
> Oh, sorry for the spam @jromero ... I realize this is still your draft. Please disregard the comments, I just put them here for our story.

No worries. I've "donated" it as I don't have the bandwidth. I believe @ameyer-pivotal said he might adopt it.

**Comment by jromero** [comment]
Closing this in favor of #85

**Pull request closed without merging**